src/imp_std.rs

```diff
@@ -6,22 +6,18 @@
 use std::{
     cell::{Cell, UnsafeCell},
     marker::PhantomData,
+    mem::MaybeUninit,
```

Comment:
THe std impl should not change, because we need to maintain 1.31 MSRV with only std feature enabled

Reply:
@matklad why we need to maintain 1.31 for `std`? It's a principal position on MSRV or any other arguments are involved?

Reply:
Two things:

*First,* it is my personal position that foundational utility crates should be pretty conservative with MSRV, where for me "pretty conservative" is something like 10-ish Rust releases.

*Second,* we've already [documented](https://docs.rs/once_cell/1.3.1/once_cell/#minimum-supported-rustc-version) that the strategy for this crate is conservative, and we can't back it out.